appendix.ru.md

```diff
@@ -1,35 +1,35 @@
-Ричард М. Столлман имеет историю плохого обращения с женщинами, заставляя их чувствовать себя некомфортно, небезопасно и нежелательно. Информацию об инцидентах, связанных с RMS и MIT, см. в [0].
+Ричард М. Столлман плохо обращался с женщинами, заставляя их чувствовать себя некомфортно, небезопасно и невостребованно. Информацию об инцидентах, связанных с Ричардом М. Столлманом и MIT, см. в [0].
```

Comment:
I've read this without looking at the English source, and the first sentence sounds really off here. "обращался с женшиными" has romantic/courting connotations in Russian so the phrase along with "невостребованно", meaning "unwanted", reads like RMS simply wasn't trying hard enough at making romance to women. Which is probably the opposite of what the text is trying to convey.

The word "unwelcome" here is hard to translate, as a direct translation "незваный", "непрошеный" is mostly used specifically towards visiting guests and doesn't sound right in a more general sense. It could be replaced with something along the lines of "their participation wasn't desirable", but it's easier to just drop it and let "uncomfortable" and "unsafe" bear the load.

Suggested variant:

"Ричард М. Столлман известен своим негативным отношением к женщинами, заставляя их чувствоваться себя некомфортно и уязвимо".